cluster/gce/gci/configure-helper.sh

```diff
@@ -3019,30 +3019,37 @@ EOF
     fi
   fi
   cat > "${config_path}" <<EOF
+version = 2
+# Kubernetes requires the cri plugin.
```

Comment:
based on the containerd/containerd config for the same case
https://github.com/containerd/containerd/blob/261c107ffc4ff681bc73988f64e3f60c32233b37/contrib/gce/configure.sh

Reply:
excellent!

Reply:
this will work with 1.4.x as well. also it's about time we moved to newer version of config.toml !!